### DISCUSSION TITLE
introduction

### DISCUSSION THREAD

**Discussion by Shulamite54** [camicroscope/caMicroscope#689] [General]
Hello team, good morning to the mentors and and my fellow contributors, hope we are all doing great.

My name is Precious, my initial application was approved my outreachy, i am so excited to contribute the caMicroscope, i am a newbie, please i need your guidance and support. 

Where and how do i start contributing, thank you.

**Comment by nwanduka** [comment]
Hi Precious @Shulamite54. Glad to have you here. For starters, I'll advise you go through the the two [open caMicroscope projects](https://www.outreachy.org/outreachy-june-2024-internship-round/communities/camicroscope/#accessibility-testing-and-improvement-for-biomedic) on the Outreachy website. Choose one that aligns with your skillset and that you can contribute to.

Then, check out [this document](https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md). It highlights various ways that you could contribute to the projects.

If you have further questions, please don't hesitate to ask.
